katalog/gatekeeper/monitoring/alert-rules.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+# Custom set of rules developed by SIGHUP to alert when Gatekeeper webhooks are misbehaving
+
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: gatekeeper-webhook-rules
+spec:
+  groups:
+    - name: GatekeeperWebhooks
+      rules:
+        - alert: GatekeeperWebhookFailOpenHigh
+          annotations:
+            message: "Gatekeeper is not enforcing {{$labels.type}} requests to the API server."
+            doc: "Gatekeeper is not enforcing {{$labels.type}} requests to the API server because the Kubernetes API server is having trouble contacting the '{{ $labels.name }}' {{$labels.type}} webhook, but the webhook is in Fail open (Ignore) mode."
+          expr: |
+            rate(apiserver_admission_webhook_fail_open_count{name=~".*.gatekeeper.sh"}[1m]) > 0
```

Comment:
I think we could increase the `rate` time window to 5m to decrease false positives.